narratives/ncov_sit-rep_2020-03-27.md

```diff
@@ -0,0 +1,2734 @@
+<br><br>
+However, the two clusters have different origins: the first cluster is most closely related to samples from China, while the second is most closely related to samples from Europe.
+<br><br>
+This shows that there have been at least two separate introductions leading to independent local transmission chains.
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 10-->
+# [Likely two local transmission chains in California](https://nextstrain.org/ncov/2020-03-27?branchLabel=clade&c=location&f_country=USA&f_division=California&label=clade:A7&p=full&r=location&m=div)
+
+We see similar patterns elsewhere in the U.S.
+<br><br>
+Here, we see several samples from Solano County, California, that are sampled on two different dates but very closely related to one another.
+<br><br>
+While the available data from California is too limited to make firm conclusions, this preliminary evidence is consistent with local transmission within Solano County.
+<br><br>
+Elsewhere in the tree, we see a similar pattern for samples in Santa Clara county.
+<br><br>
+Each county's cluster of cases is likely to due to a separate introduction, based on their relative positions in the tree.
+<!-- There is no right side text -->
+
+
+<!-- ############ SOUTH AMERICA ############# -->
+<!-- This is left-side text 11-->
+# [Updates for Central & South America](https://nextstrain.org/ncov/2020-03-27?d=tree,map&f_region=Central%20America,South%20America&p=grid&r=country&c=country&legend=closed)
+
+* [Executive summary](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=1).
+* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=2).     
+* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=3).  
+* [Updates for Europe](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=4).  
+* [Updates for North America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=7).  
+* [Updates for Central & South America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=11).  
+* [Updates for Asia](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=13).  
+* [Updates for Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=15).  
+* [Updates for Oceania](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=17).  
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=19).  
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=20).  
+
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 12-->
+# [Primarily travel-related cases sampled in South America](https://nextstrain.org/ncov/2020-03-27?c=division_exposure&d=tree,map&f_region=Central%20America,South%20America&m=div&p=full)
+We don't have much data from South America yet; of those data, most are returned travelers.
```

Comment:
```suggestion
We don't have much data from South America yet. Of those data, most are from travelers who are returning to South America from other parts of the globe.
```
It was not clear to me from the test that our data wasn't the reverse, i.e. we had data from travelers who were exposed in South America.